src/content/docs/pt-br/core-concepts/project-structure.mdx

```diff
@@ -62,12 +62,20 @@ Astro processa, otimiza e faz bundle dos arquivos de `src/` para criar o website
 
 Alguns arquivos (como componentes Astro) nem sequer são enviados ao navegador como foram escritos, mas sim são renderizados como HTML estático. Outros arquivos (como CSS) são enviados ao navegador mas podem ser otimizados e passar por bundle com outros arquivos CSS para melhorar a performance.
 
+### `src/assets`
+
+O pasta [`src/assets`](/pt-br/guides/assets/) é a pasta recomendada para armazenar arquivos (e.x. imagens) para serem processados pelo Astro. Isso não é obrigatório e ela não é uma pasta especial reservada.
+
 ### `src/components`
 
 **Componentes** são pedaços reutilizáveis de código para suas páginas HTML. Eles podem ser [componentes Astro](/pt-br/core-concepts/astro-components/) ou [componentes de frameworks de UI](/pt-br/core-concepts/framework-components/) como React ou Vue. É comum agrupar e organizar todos os componentes do seu projeto nesta pasta.
 
 Essa é uma prática comum em projetos Astro, porém não é obrigatória. Sinta-se livre para organizar seus componentes como desejar!
 
+### `src/content`
+
+A pasta `src/content/` é reservada para armazenar [coleções de conteúdo](/pt-br/guides/content-collections/) e um arquivo opicional de configuração das coleções de conteúdo.
```

Comment:
Faltou a frase final do original: "No other files are allowed inside this folder.", adicionei aqui como eu traduziria.
```suggestion
A pasta `src/content/` é reservada para armazenar [coleções de conteúdo](/pt-br/guides/content-collections/) e um arquivo opicional de configuração das coleções de conteúdo. Nenhum outro arquivo é permitido nesta pasta.
```